charts/rstudio/source/Dockerfile

```diff
@@ -27,8 +27,8 @@ RUN apt-get update && apt-get upgrade -y && apt-get install --no-install-recomme
 
 # Setup various variables
 ENV TZ="Europe/Helsinki" \
-    USERNAME="rstudio" \
-    HOME="/home/rstudio" \
```

Comment:
Why was the user dropped?

Reply:
The default user is `rstudio-server` and not `rstudio`